doc/docs/Installation.md

```diff
@@ -270,11 +270,146 @@ By default, Meep's configure script tries to guess the gcc `-march` flag for the
 Python Bindings
 ---------------
 
-A beta version of the Python bindings for Meep (PyMeep) is available in serial (i.e., non-MPI) and parallel versions. There are two ways to obtain PyMeep.
+A beta version of the Python bindings for Meep (PyMeep) is available in serial (i.e., non-MPI) and parallel versions. There are two ways to obtain PyMeep &mdash; building from source or installing binary Conda packages.
 
 ### Building From Source
 
-You can find instructions for building from source on Ubuntu for serial PyMeep [here](https://www.mail-archive.com/meep-discuss@ab-initio.mit.edu/msg05850.html), and parallel PyMeep [here](https://www.mail-archive.com/meep-discuss@ab-initio.mit.edu/msg05884.html).
+Here are instructions for building from source on Ubuntu 16.04 for serial and parallel PyMeep.
+
+#### Serial
+
+```bash
+#!/bin/bash
+
+export LD_LIBRARY_PATH="/usr/local/lib:/usr/lib/x86_64-linux-gnu/hdf5/serial"
+export LDFLAGS="-L/usr/local/lib -L/usr/lib/x86_64-linux-gnu/hdf5/serial"
+export CPPFLAGS="-I/usr/local/include -I/usr/include/hdf5/serial"
+export PYTHONPATH="$HOME/install/meep/python"
+export GUILE_WARN_DEPRECATED="no"
+
+sudo apt-get update
+sudo apt-get -y dist-upgrade
+sudo apt-get -y install libblas-dev liblapack-dev libgmp-dev libunistring-dev libmatheval-dev swig libgsl-dev libatomic-ops-dev libgc-dev libffi-dev libltdl-dev autoconf pkg-config libpng16-dev git guile-2.0-dev libhdf5-dev libfftw3-dev libpython3.5-dev python3-numpy python3-h5py
+
+mkdir ~/install
+
+cd ~/install
+git clone https://github.com/stevengj/harminv.git
```

Comment:
Probably you should build in `~/install/src` rather than directly in `~/install`